SBBDesignSystemMobileSwiftUI/Views/SBBTabView/SBBTabView.swift

```diff
@@ -53,59 +53,47 @@ public struct SBBTabView<Selection>: View where Selection: Hashable {
     private var selectionIndex: Int {
         TabBarEntryView.selectionIndex(for: selection, in: contents)
     }
-    private var contentAboveBar: Bool
```

Comment:
this means that the content is always above bar?

Reply:
Because in the app I need the content above bar. If content should be above bar, is there always a padding needed?